### PR TITLE
Frontend & Backend Integration

### LINKED ISSUE #1: Bootstrap Web Application
### Set Up Web Application:

- Create basic files for web app, and folder structure (index.html, scrips, style.css)

- Setup Maven (Configuration File pom.xml)

- Setup GCP Project (Deploy basic web app using App Engine)

- Create Initial User and Book Object

### PR DESCRIPTION
**Pull request by MauroGuerrero20**
# Features
- Integrates React Frontend and Java Servlets Backend into one service, using AppEngine's microservices.
  - Adds the ```dispatch.yaml``` configuration file to add the **backend microservice** to the project.

### Note: 
- Frontend and Backend can be deployed separately. 

## Deployment Build Instructions:

- ### Backend Repo
  - ```mvn clean; mvn package appengine:deploy```

- ### Frontend Repo
  - ```npm run build; gcloud app deploy; gcloud app deploy dispatch.yaml```

image

### CODE REVIEW

File: capstone-frontend/app.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 runtime: nodejs12
-service: reactapp # Name of Service, instead of default
+service: default
```
Comment (steven-solar) [inline capstone-frontend/app.yaml:2]: I saw you changed this to "reactapp" on the last PR, and then changed it back to "default" on this one. Just wanted to make sure this was your intention.

File: capstone-frontend/dispatch.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Rules for dispatch file...
+# Put specific cases at the top.
+dispatch:
+# Route the urls that point to the Java backend's API calls
+  - url: "*/api/*"
```
Comment (steven-solar) [inline capstone-frontend/dispatch.yaml:5]: Cool!

**Review by steven-solar — APPROVED** [review]

File: capstone-frontend/dispatch.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Rules for dispatch file...
+# Put specific cases at the top.
+dispatch:
```
Comment (antoniolinhart) [inline capstone-frontend/dispatch.yaml:3]: This is neat!

**Review by antoniolinhart — APPROVED** [review]
Looks good! This sets us up very nicely to work on the frontend and backend together!

**Comment by steven-solar** [comment]
Do you think you could add a basic test to this PR? Just have a basic servlet that sends "OK" and try fetching it from the frontend and printing out the response, just to assure that they're linked the way we believe they are.

File: capstone-frontend/dispatch.yaml (hunk 3 above)
Comment (VivekShankar) [reply capstone-frontend/dispatch.yaml:3]: +1!

**Review by VivekShankar — APPROVED** [review]
LGTM

**Pull request merged**